software/drivers/sd8887/src/wlan/mlinux/moal_cfg80211.h

```diff
@@ -201,12 +201,17 @@ int woal_cfg80211_set_rekey_data(struct wiphy *wiphy, struct net_device *dev,
 
 void woal_mgmt_frame_register(moal_private *priv, u16 frame_type, bool reg);
 void woal_cfg80211_mgmt_frame_register(struct wiphy *wiphy,
-#if CFG80211_VERSION_CODE >= KERNEL_VERSION(3, 6, 0)
+#if KERNEL_VERSION(3, 6, 0) <= CFG80211_VERSION_CODE
 				       struct wireless_dev *wdev,
 #else
 				       struct net_device *dev,
 #endif
-				       t_u16 frame_type, bool reg);
+#if KERNEL_VERSION(5, 8, 0) <= CFG80211_VERSION_CODE
```

Comment:
@majorz I guess these changes are needed from v5.7 because of https://github.com/torvalds/linux/commit/6cd536fe62ef58d7c4eac2da07ab0ed7fd19010d? If so, it would be nice to mention it on the commit log for future reference.